web/src/components/IdeToolbarNew/IdeToolbarNew.js

```diff
@@ -23,7 +24,7 @@ const IdeToolbarNew = () => {
 
   return (
     <IdeContext.Provider value={{ state, dispatch }}>
-      <div className="p-8 border-2">
+      <div className="h-full flex flex-col">
```

Comment:
Related to getting the ide container to take up the remaining space after the header has taken up it's chunk.